src/libraries/System.IO.FileSystem.Watcher/tests/Args.RenamedEventArgs.cs

```diff
@@ -16,26 +16,80 @@ public static void RenamedEventArgs_ctor(WatcherChangeTypes changeType, string d
         {
```

Comment:
My comments in FileSystemEventArgs apply for RenamedEventArgs as well.

Reply:
Done.